common/rfb/util.h

```diff
@@ -40,6 +40,8 @@ struct timeval;
 
 namespace rfb {
 
+  template <typename T> void UNUSED(const T&) {}
```

Comment:
Does LLVM support the `unused` attribute:

https://gcc.gnu.org/onlinedocs/gcc/Common-Variable-Attributes.html#Common-Variable-Attributes

That would be a lot clearer than trying to fool the compiler like this.